packages/l10n/language-information.js

```diff
@@ -6,9 +6,9 @@ import extractLanguageFromLocale from './utils/extract-language-from-locale';
 const getImportChunk = lang => {
   switch (lang) {
     case 'de':
-      return import(/* webpackChunkName: "language-data-es" */ './data/languages/es.json');
+      return import(/* webpackChunkName: "language-data-es" */ './data/languages/de.json');
```

Comment:
🙈

Reply:
It confused the living hell out of me. You go to German and get Spanish and vice versa. Like what's going on 😄. Good to "hear" from you :)

Reply:
We call it easter eggs :trollface:

Reply:
@tdeekens can you change the `webpackChunkName` as well?